src/ucp/core/ucp_ep.h

```diff
@@ -368,7 +368,7 @@ typedef struct ucp_ep {
  * Status of protocol-level remote completions
  */
 typedef struct {
-    ucs_queue_head_t              reqs;         /* Queue of flush requests which
+    ucs_hlist_head_t              reqs;         /* Queue of flush requests which
```

Comment:
it is not queue anymore (fix comment)

Reply:
in usage - it's still FIFO queue, hlist is used to save space

Reply:
i understand that, but it is list structure now, right?

Reply:
IMO type is clear from definition, but comment explains usage

Reply:
ok